torchvision/prototype/transforms/__init__.py

```diff
@@ -12,3 +12,14 @@
 from ._misc import Identity, Normalize, ToDtype, Lambda
 from ._presets import CocoEval, ImageNetEval, VocEval, Kinect400Eval, RaftEval
 from ._type_conversion import DecodeImage, LabelToOneHot
+
+# What are the migration plans for Classes without new API equivalents? There are two categories:
+# 1. Deprecated methods which have equivalents on the new API (_legacy.py?):
+# - Grayscale, RandomGrayscale: use ConvertImageColorSpace
+# 2. Those without equivalents on the new API:
+# - Pad, RandomCrop, RandomHorizontalFlip, RandomVerticalFlip, RandomPerspective, FiveCrop, TenCrop, ColorJitter,
+# RandomRotation, RandomAffine, GaussianBlur, RandomInvert, RandomPosterize, RandomSolarize, RandomAdjustSharpness,
+# RandomAutocontrast, RandomEqualize, LinearTransformation (must be added)
+# - PILToTensor, ToPILImage (_legacy.py?)
```

Comment:
Shouldn't we also have a more general `ConvertImageType` transform here that takes one of `"pil"`, `"vanilla_tensor"`, `"feature"` as input? That would follow the same approach that we have for the other conversions.

Reply:
Here is what I mean. Let's say we are close to releasing the prototype and we have achieved a significant level of BC. On day 0, we will need a PILToTensor to exist. These are likely be in a deprecated state with warnings informing the user that this functionality is removed and they should use X. Your proposal seems like a valid X alternative.

So here I think we need to figure out 2 things:
- What do we do with these legacy classes + provide their implementations on the new API (or aliases). Make sure that the classes continue to work along with the other non-deprecated classes in existing pipelines.
- Provide an alternative class that offers the new functionality in the new API.